scalactic-test/src/test/scala/org/scalactic/CatcherSpec.scala

```diff
@@ -52,7 +52,7 @@ class CatcherSpec extends UnitSpec {
   }
 
   it should "be usable as an extractor for detecting exceptions in Failed outcomes in withFixtures" in {
-    val InternalServerError = Catcher { case dbae: DBAccessException => dbae.getMessage == "500: Internal Server Error" }
+    val InternalServerError = Catcher {{case dbae: DBAccessException => dbae.getMessage == "500: Internal Server Error"}: PartialFunction[Throwable, Boolean] }
```

Comment:
**Compiler cannot infer type**
Compiler cannot infer the type of a union type parameter, the type needs to be explicitly declared.